### DISCUSSION TITLE
Why we cant play video in editor

### DISCUSSION THREAD

**Discussion by nobody4322** [GrapesJS/grapesjs#4137] [Q&A]
I have added a video block, but i cant  play video using control button. To play video i have to preview the editor then controls are working fine. is it possible to play or stop  video using controls without going to preview section??

**Comment by artf** [comment]
If you try to think about it a little bit more you'll understand that there is a simple UX issue. 
If you're able to interact with embedded content (video, iframe, etc.) that means you won't be able to select it in the canvas by simply clicking on it. 
But this is all up to you, if you want it anyway, you can do it, just explore the code of the component and extend it as you wish.
